Whac-A-Mole.md

```diff
@@ -0,0 +1,60 @@
+# Summary
+> A task to show that imitation exists in competitive settings.
+
+# References
+> Main paper: https://www.pnas.org/content/pnas/110/50/20046.full.pdf
+> 
+> Supporting paper: https://www.pnas.org/content/pnas/suppl/2013/11/20/1305996110.DCSupplemental/pnas.201305996SI.pdf
+> 
+> Movie 1: http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1305996110/-/DCSupplemental/sm01.wmv
+> 
+> Movie 2: http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1305996110/-/DCSupplemental/sm02.wmv 
+
+# Stimuli
+## The visual components
+> Players used a “rod” to reach for targets (green and yellow) that were briefly shown among distracters (red, magenta, blue, and purple) on a large touch screen
+> 
+> ![Whac1](images/Whac1.png)
+> ![Whac2](images/Whac2.png)
+> 
+> The open circle shows the rod on the screen.
+> 
+> Listed below is the meaning of each target and the x/y color–space coordinates according to the International Committee on Illumination 
+> 
+> Green: Player gained a point;  22.3 cd/m2, 0.186/0.760
+> 
+> Yellow: Opponent lost a point; 22.6 cd/m2, 0.340/0.545
+> 
+> Red: Player lost a point; 6.64 cd/m2, 0.621/0.322
+> 
+> Magenta: Opponent gained a point; 6.32 cd/m2, 0.314/0.131
+> 
+> Blue: Added to both players; 5.49 cd/m2, 0.167/0.121
+> 
+> Purple: Removed from both players; 5.22 cd/m2, 0.213/0.135
+
+## Materials for alternative versions of the experiment 
+> Players played 30 different variations of the game.
+
+# Procedure
+## Steps
+> Participants were paired to stand on either side of a large screen each holding a rod
+> 
+> They were instructed to hit the green and yellow targets on the screen as they appeared. They would accumulate points for each correct target they hit.
+> 
+> Participants played 30 Whac-A-Mole games that each lasted 40 seconds. They were allowed to take breaks between rounds
+
+## Roles 
+> Each participant had the same role.
+
```

Comment:
In Experiment Two: a confederate was instructed to adjust its behavior across blocks. The confederate was instructed to act as a slow player in block 1 (games 1–4) and block 3 (games 9–12) and act as a fast player in block 2 (games 5–8) and block 4 (games 13–16).